packages/tinacms/src/toolkit/fields/components/reference/utils/fetch-optioins-query-builder.ts

```diff
@@ -0,0 +1,22 @@
+// This type is used in collectionFilter for reference field, it represent the datatype of the field in the referenced collection that will be used to do the filter
+// This type is extendable if we need to support more type for reference selection filter (e.g. boolean)
+export type FilterValue = string[] | string
+export type CollectionFilters =
+  | Record<string, FilterValue>
+  | (() => Record<string, FilterValue>)
+
+//Currently only support in for filter, this function will loop through the record and build the filter query
```

Comment:
**question**: Does this comment add value?

I worry it will be ignored at the next refactor - see https://www.ssw.com.au/rules/code-commenting/#what-are-the-downsides-of-comments

Reply:
This comment is not a TODO, it is a explanation of what the below function means, thus it does not require a PBI link to it. Also i have reword this comment 😊

Reply:
Done - As per our conversation , the comment does not bring much value, thus comment removed